CMakeLists.txt

```diff
@@ -145,6 +102,8 @@ endif()
 addtest(test0016 tests/test_0016-finish-getitem-for-rawarray.cpp)
 addtest(test0019 tests/test_0019-use-json-library.cpp)
 addtest(test0030 tests/test_0030-recordarray-in-numba.cpp)
+addtest(test0293_cuda_alloc tests/test_0293-cuda-allocation-tests.cpp)
+addtest(test0293_listarray_num tests/test_0293-listarray-num-kernel.cpp)
```

Comment:
Could we, for now, just comment these out? Or create a separate `Makefile` linking the awkward libraries to the test? You did say, `C++` isn't where we are going to test these kernels out. So should I just comment them out, in case any one of us wanted to use it for debugging purposes?

Reply:
You can just leave them out entirely. They'll find a new home in the unit tests you're developing. In fact, since the new unit tests won't need to be dependent on CMake/CTests, perhaps the full path to the library can be filled in with `sed` or something.

The final tests will be in Python (hence, no full path necessary because it will go through the `startup.cpp`/callback mechanism), but those Python tests can come after this PR.

(Also, when you finish any of these inline tasks, just "resolve conversation" so that I know it's done.)

Reply:
Okay, I'll completely remove them from the `CMakeLists.txt`. I am a bit lost here, which `unit` tests are you referring to here? Is it the `test0293_*`?

Reply:
The new ones you're writing (maybe next week) that might run on AWS.

Reply:
Okay, that makes sense!